sympy/matrices/immutable.py

```diff
@@ -158,6 +158,8 @@ def __setitem__(self, *args):
 
     subs = MatrixBase.subs
 
+    xreplace = MatrixBase.xreplace
+
```

Comment:
I don't think this change is tested, is it? You seem to only test a `Matrix` below. Can you add a test for this as well?